base/methodshow.jl

```diff
@@ -253,18 +267,20 @@ function show_method_list_header(io::IO, ms::MethodList, namefmt::Function)
                     "builtin function"
                : # else
                     "generic function")
-        print(io, " for ", what, " ", namedisplay)
+        print(io, " for ", what, " ", namedisplay, " from ")
+        printstyled(io, ms.mt.module, color=:blue)
     elseif '#' in sname
         print(io, " for anonymous function ", namedisplay)
     elseif mt === _TYPE_NAME.mt
         print(io, " for type constructor")
     else
         print(io, " for callable object")
     end
-    n > 0 && print(io, ":")
-    nothing
+    !iszero(n) && print(io, ":")
 end
 
+const METHODLIST_MODULECOLORS = [:cyan, :green, :yellow]
```

Comment:
Why does this have a separate and different list of colors than stack traces?

Reply:
I'll unify the colors between these and the stack traces :+1: